Source/WPEFramework/SystemInfo.cpp

```diff
@@ -35,6 +35,7 @@ namespace PluginHost {
         , _time(nullptr)
         , _provisioning(nullptr)
         , _flags(0)
+        , _type(END_LIST)
```

Comment:
I think I am missing something? What goal does this _type serve ?

Reply:
Just to get the current event type fired. From the callback will use this type and updating the accessor details

Reply:
Moved this logic to PluginServer.h as per discussion